javascript/javascript3/week1/a-cool-api.js

```diff
@@ -0,0 +1,14 @@
+//"Bored" is a free API to find something to do by getting suggestions for random activities
+
+const myActivity = document.createElement("p");
```

Comment:
This is an interesting API. I'll have to remember this next time I'm thinking of playing a game of "finding a movie to watch on netflix for 20 minutes until I give up". 
That being said, you should really put your paragraph in the HTML file because creating it here is unnecessary.